odoo/addons/base/models/ir_qweb.py

```diff
@@ -430,7 +430,7 @@
 RSTRIP_REGEXP = re.compile(r'\n[ \t]*$')
 VARNAME_REGEXP = re.compile(r'^[A-Za-z_][A-Za-z0-9_]*$')
 # Attribute name used outside the context of the QWeb.
-SPECIAL_DIRECTIVES = {'t-translation', 't-ignore', 't-title'}
+SPECIAL_DIRECTIVES = {'t-translation', 't-ignore', 't-title', 't-forbid-sanitize'}
```

Comment:
`t-forbid-sanitize` is not a special, the directive is linked with the `t-snippet`

Special directives are ones that qweb not take care. Here it is used by a directive.